tests/chainer_tests/training_tests/triggers_tests/test_interval_trigger.py

```diff
@@ -116,6 +116,14 @@ def test_resumed_trigger_backward_compat(self):
                 trainer.updater.update()
                 self.assertEqual(trigger(trainer), expected)
 
+    def test_str(self):
+        trigger = training.triggers.IntervalTrigger(*self.interval)
+
+        expected = 'IntervalTrigger(%d, \'%s\')' % tuple(self.interval)
+        actual = str(trigger)
+
+        self.assertEqual(expected, actual)
```

Comment:
```suggestion
        assert expected == actual
```

nit-picking following [how to write tests (contribution guide)](https://docs.chainer.org/en/stable/contribution.html#how-to-write-tests).

Reply:
I didn't know that clause in the guide, thanks for pointing out.

I really hate assert statements without any error messages, so I'll push that change with an error message.